students/slawinski_amadeusz/lesson_01_basics/1.01_sum_of_three_numbers.py

```diff
@@ -0,0 +1,6 @@
+# This program reads three numbers and prints their sum:
+a = int(input())
```

Comment:
Jedno literkowe nazwy zmiennych są mało czytelne. Nie sugeruj się templatem z Snakify, niekiedy są w nich błędy.
Zapoznaj się z PEP8 (zwłaszcza nazywanie zmiennych), jest to przewodnik po code convention w Pythonie. Link jest w 'Reference Materials' w opisie kursu (plik redme).
Również można by dodać jakiś komunikat, żeby użytkownik wiedział czego się od niego wymaga.
Komunikat możesz przekazać jako parametr funkcji input: `input("Podaj liczbę:)`